docs/process/graduation.md

```diff
@@ -0,0 +1,27 @@
+Graduating Components from Experimental
+=======================================
+
+This document contains a checklist for the requirements to migrate a component from experimental.
+
+- [ ] The crate should fully conform with the [ICU4X Style Guide](style_guide.md):
+  - [ ] The crate should have a complete library header as shown in [boilerplate.md](boilerplate.md) with Clippy passing
```

Comment:
Also something about Cargo.toml:

 - Must use license, not license-file
 - useful description
 - correct repo link, authors, categories, include
 - correct docs.rs features settings (and cargo-all-features)
 - Audit all features so that any `foo/bar` in a feature is `foo?/bar` when `foo` is an optional dep, unless you intend to enable `foo` (in which case, write two entries)

Reply:
* Use `dep:` for dependencies